cirq-core/cirq/contrib/quimb/state_vector.py

```diff
@@ -177,6 +177,8 @@ def tensor_expectation_value(
     if ram_gb > max_ram_gb:
         raise MemoryError(f"We estimate that this contraction will take too much RAM! {ram_gb} GB")
     e_val = tn.contract(inplace=True)
+    if isinstance(e_val, qtn.TensorNetwork):
+        e_val = e_val.item()
```

Comment:
AFAICT, the contraction above will always produce a scalar, so why not call `item` unconditionally?

```python
e_val = tn.contract(inplace=True).item()
```

Reply:
With quimb-1.4.2 `tn.contract()` may return a standard float which does not have the `item` method.
We could do `e_val = np.asarray(tn.contract(...)).item()`, but that is IMO harder to parse for humans than above. :)

We could also require `quimb==1.5`, but I feel that is a bit imposing on anyone installing cirq.